### PR TITLE
Rubocop: Your move, Creep

### PR DESCRIPTION
**Pull request by jaredmoody**
Low hanging rubocop fixes for #945

I’m not 100% sure on the Lint/EmptyWhen fix if someone can double check that, I’d appreciate it.

### CODE REVIEW

File: shoes-core/spec/shoes/framework_learning_spec.rb [hunk 1]
```diff
@@ -18,8 +18,10 @@ def gui_init
   end
 end
 
-class Learning::App
-  include TextPlugin::App
+class Learning
+  class App
+    include TextPlugin::App
+  end
```
Comment (PragTob) [inline shoes-core/spec/shoes/framework_learning_spec.rb]: This seems to break the tests, cause Learning is a `module` not a class :)

File: shoes-core/lib/shoes/common/positioning.rb [hunk 2]
```diff
@@ -17,7 +17,7 @@ def _position(left, top)
         gui.update_position if gui && gui.respond_to?(:update_position)
       end
 
-      # displace(left: a number, top: a number) » self
+      # displace(left: a number, top: a number) >> self
```
Comment (PragTob) [inline shoes-core/lib/shoes/common/positioning.rb]: :+1:

File: samples/expert-pong.rb [hunk 3]
```diff
@@ -38,8 +38,10 @@
 
     # move the @comp paddle, speed based on `compuspeed` variable
     @comp.left +=
-      if nx + (ball_diameter / 2) > @comp.left + paddle_size; compuspeed
-      elsif nx < @comp.left; -compuspeed
```
Comment (PragTob) [inline samples/expert-pong.rb]: wow that was ugly :D

File: samples/expert-irb-adjusted.rb [hunk 4]
```diff
@@ -44,10 +44,7 @@ def run(str)
     $stdout.rewind
     [output, obj]
   rescue Object => e
-    case e
-    when Empty, Continue
-    else @line = ""
-    end
+    @line = "" unless e == Empty || e == Continue
```
Comment (jaredmoody) [inline samples/expert-irb-adjusted.rb]: Rubocop was complaining about the empty `when`, but I made the change here - we're through all the auto-correctable cops now. I wasn't sure about this change though, so I mentioned it in the PR comments.  Thanks for catching that I should have used `===`. How do you want this to look though, would this be better?

`@line = "" unless [Empty, Continue].include?(e.class)`

**Comment by PragTob** [comment]
Rubocop bumms out... 

```
samples/expert-irb-adjusted.rb:47:25: C: Avoid the use of the case equality operator ===.

    @line = "" unless e === Empty || e === Continue

                        ^^^

samples/expert-irb-adjusted.rb:47:40: C: Avoid the use of the case equality operator ===.

    @line = "" unless e === Empty || e === Continue

                                       ^^^

samples/expert-irb.rb:47:25: C: Avoid the use of the case equality operator ===.

    @line = "" unless e === Empty || e === Continue

                        ^^^

samples/expert-irb.rb:47:40: C: Avoid the use of the case equality operator ===.

    @line = "" unless e === Empty || e === Continue

                                       ^^^
```

I guess we could substitute it through an `e.is_a?(clazz)`

**Comment by jaredmoody** [comment]
OK, used `is_a?` to fix up that exception comparison and fixed that module definition.

**Comment by PragTob** [comment]
Thank you!

**Pull request merged**